account_analytic_parent/models/__init__.py

```diff
@@ -1,8 +1,4 @@
 # -*- coding: utf-8 -*-
-# Copyright 2017 Eficent Business and IT Consulting Services S.L.
-# Copyright 2017 Luxim d.o.o.
-# Copyright 2017 Matmoz d.o.o.
-# Copyright 2017 Deneroteam.
-# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
```

Comment:
Because `__init__.py` are not code by themselves, and making light these files is better to maintain.